packages/modules/localstack/src/localstack-container.test.ts

```diff
@@ -69,4 +69,18 @@ describe("LocalStackContainer", () => {
     await container.stop();
     await awsCliInDockerNetwork.stop();
   });
+
+  it("should use access environment variables and default creds", async () => {
+    const container = await new LocalstackContainer()
+      .withEnvironment({
+        DEFAULT_REGION: "eu-west-3",
+      })
+      .start();
+
+    expect(container.getRegion()).toBe("eu-west-3");
+    expect(container.getAccessKey()).toBe("test");
+    expect(container.getSecretKey()).toBe("test");
```

Comment:
The container is mapping the environment variables to these getter methods, but this test is not checking that LS is actually using them. From the deprecation notices in the docs I get the impression they're not being used.

Reply:
Yes, I think you're right... I see nothing in current documentation referencing these... Here again, I tried to reproduce the behavior of the test containers-java/local stack module, but this may be inappropriate.